packages/firebase_auth/firebase_auth_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.2
```

Comment:
Make this a patch version increase (0.1.1+1)

Reply:
I disagree, but we can discuss this in the post-mortem.